client/my-sites/sidebar/sidebar-menu-item.jsx

```diff
@@ -21,8 +22,11 @@ var SidebarMenuItem = React.createClass( {
 		buttonLabel: React.PropTypes.string,
 		onNavigate: React.PropTypes.func,
 		icon: React.PropTypes.string,
+		preload: React.PropTypes.string
```

Comment:
Could we make this name clearer? It seems a bit ambiguous, as if it could be _a function_ to run when preloading, _a boolean_ to indicate whether or not to preload, etc… I actually wouldn't suspect it would be a string specifying which section to preload given the name